### DISCUSSION TITLE
[Emm]

### DISCUSSION THREAD

**Discussion by 18611328786Xu** [WeNeedHome/SummaryOfLoanSuspension#369] [General]

**Comment by yalo-ddaa** [comment]
差不多得了，墙是单向的不懂吗？
你看不了google，但外国人一直都能上百度。

**Reply by 18611328786Xu** [reply]
> 差不多得了，墙是单向的不懂吗？ 你看不了google，但外国人一直都能上百度。

唉。

**Comment by ChinaChenMingQuan** [comment]
还挺讽刺的，国内的事情居然在国外才能讨论。

**Reply by kennard520** [reply]
中国领导人不人说

**Reply by 18611328786Xu** [reply]
> 中国领导人不人说

其实我觉得真正的高层领导不见得知道😂

**Reply by kennard520** [reply]
这么大的事不知道？ 那他就有点失职了，肯定知道，维稳了

**Comment by dccchao** [comment]
哑巴人民 哈哈哈哈哈哈 我们能怎么办呢

**Reply by PaddySu** [reply]
意思挨打了不能喊？

**Reply by kennard520** [reply]
跟谁喊啊，喊了继续打

**Reply by 18611328786Xu** [reply]
> 跟谁喊啊，喊了继续打

几代人的全部资产要是买个烂尾楼，估计杀人的心都有，不能不让人家说说。何况中国人其实已经非常老实了。

**Reply by kennard520** [reply]
上次河南400亿，几千人去维权不是被打了么，生在中国，没法，就是这个样子，所以大家要团结起来

**Comment by NKNUAL** [comment]
怎么丢人？

**Reply by 18611328786Xu** [reply]
> 政府丢人呀。

**Reply by ppwq2022** [reply]
> > 政府丢人呀。
ZF倒没啥丢人的，搁哪国都有这样的问题，但相关监管部门管理渎职和监管缺失是无法推卸的

**Reply by NKNUAL** [reply]
做出了这样事就别想要脸面，对你来说是在国外丢脸了，但是对这群买了烂尾楼的人来说呢，不要你感同身受，但至少不是冷嘲热讽吧

**Comment by weizhanzhan** [comment]


**Reply by 18611328786Xu** [reply]
> 你觉得这是丢人？

我觉得是政府比较丢人

**Reply by weizhanzhan** [reply]
不止此事，不止现在

**Comment by fanjumin** [comment]
脑残

**Reply by 18611328786Xu** [reply]
> 脑残

那我祝你买个烂尾吧。

**Comment by milaixian111** [comment]
傻逼你才是这里最丢人的那个

**Reply by 18611328786Xu** [reply]
> 傻逼你才是这里最丢人的那个

那就祝你买个烂尾楼吧

**Comment by MatthewDevelop** [comment]
只要是事实,该丢人那就得丢

**Reply by 18611328786Xu** [reply]
> 只要是事实,该丢人那就得丢

好多事情都是不闹大不解决

**Reply by cy771** [reply]
怎么一直没更新了？

**Comment by cy771** [comment]
怎么一直没更新了？

**Reply by Gingko30** [reply]
可能喝茶去了😭

**Reply by cy771** [reply]
唉，注意安全
